actionview/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Only clear ActionView cache in development on file changes
+
+    To speed up development mode, view caches are only cleared when files in
+    the view paths have changed. Applications which have implemented custom
+    ActionView::Resolver subclasses may need to add their own cache clearing.
```

Comment:
I was wondering if we should expose `ActionView::LookupContext::DetailsKey.clear` as `clear_cache` on resolvers. iirc DetailsKey is private so users shouldn't know about that constant directly — and I don't know how else they'd clear the cache. What do you think?

Reply:
👍 I agree it needs a better (and not private) name. It's a little tricky because having a `Resolver#clear_cache` doesn't currently make sense, when testing I got errors unless we cleared all the caches.

Maybe we should expose `ActionView::CacheExpiry.clear_cache` (ie. make it class-level and document it)

Reply:
Should the resolvers know about `ActionView::CacheExpiry`? Currently the cache expiry knows about the resolvers, so it seems off that we allow double "binding". Since it's just abstracting a reference to DetailsKey anyway, how about a class-level `ActionView::Resolver.clear_caches`?

Which we then also use in lieu of `clear_cache` in CacheExpiry.